contracts/payloads/IGP20/PayloadIGP20.sol

```diff
@@ -0,0 +1,550 @@
+    |     Proposal Payload Actions      |
+    |__________________________________*/
+
+    /// @notice Action 1: Deploy weETH/USDC vault and set related configs.
+    function action1() internal {
+        deploy_weETH_USDC_VAULT();
+    }
+
+    /// @notice Action 2: Deploy weETH/USDT vault and set related configs.
+    function action2() internal {
+        deploy_weETH_USDT_VAULT();
+    }
+
+    /***********************************|
+    |     Vault Deployment Helper       |
+    |__________________________________*/
+    function deploy_weETH_USDC_VAULT() internal {
+        // Deploy weETH/USDC vault.
+        address vault_ = VAULT_T1_FACTORY.deployVault(
+            address(VAULT_T1_DEPLOYMENT_LOGIC),
+            abi.encodeWithSelector(
+                IFluidVaultT1DeploymentLogic.vaultT1.selector,
+                WEETH_ADDRESS, // weETH,
+                USDC_ADDRESS // USDC
+            )
+        );
+
+        // Set user supply config for the vault on Liquidity Layer.
+        {
+            AdminModuleStructs.UserSupplyConfig[]
+                memory configs_ = new AdminModuleStructs.UserSupplyConfig[](1);
+
+            configs_[0] = AdminModuleStructs.UserSupplyConfig({
+                user: address(vault_),
+                token: WEETH_ADDRESS,
+                mode: 1,
+                expandPercent: 25 * 1e2, // 25%
+                expandDuration: 12 hours,
+                baseWithdrawalLimit: 2_500 * 1e18 // 2_500 weETH
+            });
+
+            LIQUIDITY.updateUserSupplyConfigs(configs_);
+        }
+
+        // Set user borrow config for the vault on Liquidity Layer.
+        {
+            AdminModuleStructs.UserBorrowConfig[]
+                memory configs_ = new AdminModuleStructs.UserBorrowConfig[](1);
+
+            configs_[0] = AdminModuleStructs.UserBorrowConfig({
+                user: address(vault_),
+                token: USDC_ADDRESS,
+                mode: 1,
+                expandPercent: 20 * 1e2, // 20%
+                expandDuration: 12 hours,
+                baseDebtCeiling: 7_500_000 * 1e6, // 7.5M
+                maxDebtCeiling: 20_000_000 * 1e6  // 20M
+            });
+
+            LIQUIDITY.updateUserBorrowConfigs(configs_);
+        }
+
+        // Update core settings on weETH/USDC vault.
+        {
+            IFluidVaultT1(vault_).updateCoreSettings(
+                100 * 1e2, // 1x     supplyRateMagnifier
+                100 * 1e2, // 1x     borrowRateMagnifier
+                75 * 1e2, // 75%     collateralFactor
+                80 * 1e2, // 80%    liquidationThreshold
+                90 * 1e2, // 90%    liquidationMaxLimit
+                5 * 1e2, // 5%     withdrawGap
+                3 * 1e2, // 3%     liquidationPenalty
+                0 // 0%     borrowFee
+            );
+        }
+
+        // Update oracle on weETH/USDC vault.
+        {
+            IFluidVaultT1(vault_).updateOracle(
+                0x67e40d401286f21e92adbf8DB2D392a1af735A95
```

Comment:
Wrong oracle address.

Reply:
I sent the wrong one, my bad. This is the correct address: [0xA9271615297eF2b471f6d87617b0452773d7651C](https://etherscan.io/address/0xA9271615297eF2b471f6d87617b0452773d7651C)

Reply:
Fixed in https://github.com/Instadapp/inst-governance/pull/18/commits/1c00c33598a09fe6300c5f35d21dd18c9bcad46b